### DISCUSSION TITLE
Trouble with timestamp signature

### DISCUSSION THREAD

**Discussion by Nass-cmd** [Keyfactor/signserver-ce#41] [Q&A]
Hello,
      I have configured a timestamp worker and pdfSigner in signserver.they work, but  when verifying signature it shows:" signature includes an embbed timestamp but could not be verified." I am using signerserver ce 5.2 and adobe for verification.How can i fix this?

**Comment by netmackan** [comment]
Is you timestamp worker configured with a time-stamp signer certificate that is issued by a CA that is trusted in your Adobe Reader?
If you use certificates from your own CA to issue the time-stamp signer certificate then probably you need to make that CA (or its root CA) trusted in Adobe Reader.

**Reply by nicoveiguela** [reply]
Does any one know where I can buy a Certificate trusted in adobe?

**Comment by Nass-cmd** [comment]
My certificat is self signed,so what I do is that I make adobe trust my
certificat by adding to its trust store.Thank you

Le mer. 26 avr. 2023 à 10:40, Markus Kilås ***@***.***> a
écrit :

**Reply by nicoveiguela** [reply]
Thanks for you answer. How did you add your certificate to the trust store? If you can share, thanks again
